addons/windows_95_theme/win95_plugin.gd

```diff
@@ -15,3 +15,11 @@ func _enter_tree():
 func _exit_tree():
 	# Clean-up of the plugin goes here.
 	pass
+
+func _disable_plugin():
+	var interface: EditorInterface = get_editor_interface()
+	var settings: EditorSettings = interface.get_editor_settings()
+	settings.set("interface/theme/preset", "")
+	settings.set("interface/theme/base_color", Color.SILVER)
```

Comment:
I want to see this PR merged.

But this is probably the wrong base color. I think you forgot to change it. The correct color is `#363d4a`.

Reply:
I have no intent to restart Godot ... maybe @johnweven can fix it while merging?